### ISSUE TITLE
Cant Enter Limbus.

### ISSUE THREAD

**Issue by Zeldha** [LegionDark/Issues#1337]
**Date & Time**:  January 8th, 2017

**FFXI Client Version (use `/ver`)**:  30161227_1

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**:  yes

**Character Name**:  Vida

**Nation**:  San d'Oria

**Job(level)/Sub Job(level)**:  THF 99/ DNC 49

**NPC or Monster or item Name**: 

**Zone name**:  Limbus (Appolyon)

**Coordinates (use `$where`)**:  

**ffxiah.com link (for items issues only)**:  -

**Multi-boxing? (multiple clients on same connection)**:  not at the moment.

**Steps To Reproduce / any other info**:  Cant enter limbus. Key Items ready: Black Card, Cosmos Cleanse.
SS attached.
![vida_2017 01 28_094527](https://cloud.githubusercontent.com/assets/16524259/22393392/81a3d538-e540-11e6-8ccf-50f6fb093cf9.png)
![vida_2017 01 28_095912](https://cloud.githubusercontent.com/assets/16524259/22393393/81ab87ec-e540-11e6-8f2c-48fc4cda3722.png)

**Comment by TeoTwawki** [comment]
which limbus? there are nearly dozen limbus's! multiple in applyon multiple in temenos.

If its not the one I disabled deliberately (crashing mimic floor) its a dsp prob I can't do anything about anyway. 😢

**Comment by Zeldha** [comment]
Cant Enter Apollyon EAST, CAN enter Apollyon WEST.

**Comment by TeoTwawki** [comment]
Thanks will try and see why then make sure someone who can potentially fix it knows.


###### gawd I hate limbus code

**Comment by TeoTwawki** [comment]
crap. theres 2 west ones. the one disabled due to a crash is south-west apollyon. does northwest let you enter?

**Comment by Raelious** [comment]
Yes Northwest does let you enter, I farm that zone pretty regularly. Northeast is disabled from there though.
